admin/tests/unit/services/request-manager-test.js

```diff
@@ -21,8 +21,8 @@ module('Unit | Service | request-manager', function (hooks) {
     currentDomainService = this.owner.lookup('service:currentDomain');
     sinon.stub(currentDomainService, 'isFranceDomain').value(false);
 
-    intlService = this.owner.lookup('service:intl');
-    sinon.stub(intlService, 'primaryLocale').value('fr');
+    localeService = this.owner.lookup('service:locale');
+    sinon.stub(localeService, 'currentLocale').value('fr');
   });
```

Comment:
Dans ce test unitaire, c'est `acceptLanguageHeader` que l'on souhaite mocker. Et on a plus besoin du `currentDomainService`

```suggestion
    localeService = this.owner.lookup('service:locale');
    sinon.stub(localeService, 'acceptLanguageHeader').value('fr');
  });
```

Reply:
Le commentaire n'a pas été pris en compte ;)

Reply:
Ah oui, zut j’avais bien vu le point _« plus besoin du currentDomainService »_ mais le point _« c'est acceptLanguageHeader que l'on souhaite mocker »_ m’avais échappé. 

C'est modifié cette fois.